### PR TITLE
set plugin group to web if not specified, fixes #390

### LINKED ISSUE #390: option -t not working on develop
I cloned a fresh repo, and tried to run following command.
```bash
./owtf.py -t active http://127.0.0.1

 _____ _ _ _ _____ _____
|     | | | |_   _|   __|
|  |  | | | | | | |   __|
|_____|_____| |_| |__|

        @owtfp
    http://owtf.org
    
Current Path: ./owtf.py
Syntax: owtf.py [ options ] <target1 target2 target3 ..> where target can be: <target URL / hostname / IP>
                    NOTE: targets can also be provided via a text file


Examples:

Run all web plugins:                         owtf.py http://my.website.com
Run only passive + semi_passive plugins:             owtf.py -t quiet http://my.website.com
Run only active plugins:                     owtf.py -t active http://my.website.com

Run all plugins except 'OWASP-CM-001: Testing_for_SSL-TLS': owtf.py -e 'OWASP-CM-001' http://my.website.com
Run all plugins except 'OWASP-CM-001: Testing_for_SSL-TLS': owtf.py -e 'Testing_for_SSL-TLS' http://my.website.com

Run only 'OWASP-CM-001: Testing_for_SSL-TLS':             owtf.py -o 'OWASP-CM-001' http://my.website.com
Run only 'OWASP-CM-001: Testing_for_SSL-TLS':             owtf.py -o 'Testing_for_SSL-TLS' http://my.website.com

Run only OWASP-IG-005 and OWASP-WU-VULN:             owtf.py -o 'OWASP-IG-005,OWASP-WU-VULN' http://my.website.com
Run using my resources file and proxy:             owtf.py -m r:/home/me/owtf_resources.cfg -x 127.0.0.1:8080 http://my.website.com

Run using TOR network:                    owtf.py -o OWTF-WVS-001 http://my.website.com --tor 127.0.0.1:9050:9051:password:1

Run Botnet-mode using miner:                    owtf.py -o OWTF-WVS-001 http://my.website.com -b miner

Run Botnet-mode using custom proxy list:                  owtf.py -o OWTF-WVS-001 http://my.website.com -b list:proxy_list_path.txt

ERROR: Invalid Plugin Type 'active' for Plugin Group 'None'. Valid Types: 
```

### PR DESCRIPTION
**Pull request by DarKnight--**
This PR sets the plugin group to web if not specified. This makes -t command functional.

## Description

Set the plugin_group to web if not specified.

## Related Issue

#390 

## Reviewers

@delta24 

## How Has This Been Tested?

./owtf.py -t passive

## Screenshots (if appropriate):

screenshot from 2016-04-14 11 33 19

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [ ] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: owtf.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def process_options(user_args):
 
     # Default settings:
     profiles = {}
-    plugin_group = arg.PluginGroup
+    plugin_group = arg.PluginGroup or 'web'
```
Comment (delta24) [inline owtf.py]: Add a comment here which explains the "or" statement :)

Comment (DarKnight--) [inline owtf.py]: If the command line option -g is passed the plugin_group will be set to the specified group, else `None` is set in `arg.PluginGroup`. In the case if no group is specified we set it to `web` to make `-t ` functional with `web` plugin_group. The default is set to `web` as discussed in issue #390 .

Comment (delta24) [inline owtf.py]: I meant in the code. xD

**Comment by DePierre** [comment]
I tested the changes and it seems to properly fix #390, although #375 is still unfixed.

**Pull request merged**